.gitattributes

```diff
@@ -200,4 +200,12 @@ makefile       text
 ## EXECUTABLES
 *.pyc binary
 
-
+## LINGUST OVERRIDES
+/Dockerfile.dev linguist-language=dockerfile
+/docker-compose linguist-language=dockerfile
```

Comment:
docker-compose files are already styled, so docker-compose related lines can be removed.